OsmAnd/src/net/osmand/plus/wikivoyage/data/TravelObfHelper.java

```diff
@@ -1112,6 +1119,14 @@ public boolean publish(Amenity amenity) {
 												String tag = key.replaceFirst(GPX_EXTENSION_TAG_PREFIX, "");
 												String val = amenity.getAdditionalInfo(key);
 												gpxFileExtensions.put(tag, val);
+											} else if (key.startsWith(POINTS_GROUPS_TAG_PREFIX)) {
+												final String DELIMITER = "~~~";
+												String joinedValues = amenity.getAdditionalInfo(key);
+												List<String> values = Arrays.asList(joinedValues.split(DELIMITER));
+												if (key.endsWith("names")) pgNames.addAll(values);
```

Comment:
formatting

Reply:
formatting is OK here (this is internal class instance in method parameters so formatting is looking sligtly weird)